en/includes/guides/branding/add-custom-layouts.md

```diff
@@ -0,0 +1,144 @@
+    
+### Step 1: Create the extensions folder
+
+Navigate to `webapps/authenticationendpoint` and create a new folder named `extensions`.
+
+!!! note
+    Repeat this step for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint`.
+
+### Step 2: Create the basic custom layout resources
+
+To create the basic custom layout resources:
+
+1. In the `authenticationendpoint` folder, navigate to the created `extensions` folder and create the `layouts` and `custom` folders as follows.
+    
+    ```
+    └── extensions
+        └── layouts
+            └── custom
+    ```
+
+3. Navigate to the `webapps/authenticationendpoint/includes/layouts/default` folder, copy the `body.html` file, and add it to the created `custom` folder.
+
+4. Create the following files and the folder inside the `custom` folder:
+    - `styles.css` file
+    - `script.js` file
+    - `assets` folder
+
+!!! note
+    Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint`.
+
+### Step 3: Add a custom layout design
+
+To add a custom layout to the login page:
+
+1. Navigate to `webapps/authenticationendpoint/extensions/layouts/custom/` and follow the instructions provided below:
+
+    1. Copy the code snippet from the [html file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/body.html) into the `body.html` file.
+
+    2. Copy the code snippet from the [css file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/style.css) into the `styles.css` file.
+
+    3. Copy the code snippet from the [javascript file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/script.js) into the `script.js` file.
+
+    4. Copy the [illustration.svg](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/illustration.svg) file into the `assets` folder.
+
+2. Navigate to the `webapps/authenticationendpoint/includes` folder and do the following:
+
+    1. Copy content from the [`jsp` file containing the project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/project-title-1.jsp) to the `webapps/authenticationendpoint/extensions/product-title.jsp` file of the app.
+
+    2. Copy content from the [`jsp` file containing the project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-1.jsp) to the `webapps/authenticationendpoint/extensions/product-footer.jsp` file of the app.
+
+
+    !!! note
+        Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well. You will be copying the following files:
+        
+
+        - [Project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-title-2.jsp).
+
+        - [Project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-2.jsp)
```

Comment:
This step 2 is irrelevant for IS-7.0. In 7.0 users can modify the header and footer via branding configurations. Should point this to there.